src/LP_API.js

```diff
@@ -1,29 +1,27 @@
-import Symbol from 'es6-symbol'
```

Comment:
In the changelog for this version change I'd imagine we'll note that `IE6` support is now dropped?

Reply:
Good point! I'll add that to the release notes. We typically don't maintain changelog files on client-side libs.